JournalMonitor/JournalMonitor.cs

```diff
@@ -2594,7 +2594,7 @@ async void ModuleArrived()
                                     decimal total = JsonParsing.getDecimal(data, "Total");
                                     bool full = EDDI.Instance.CurrentShip?.fueltanktotalcapacity == null
                                         ? false
-                                        : Math.Round(total) == EDDI.Instance.CurrentShip.fueltanktotalcapacity;
+                                        : total == (EDDI.Instance.CurrentShip.fueltanktotalcapacity ?? 0M);
```

Comment:
Floating point `==` is vulnerable to underflow errors. I recommend instead:
`Math.Abs(total - (EDDI.Instance.CurrentShip.fueltanktotalcapacity ?? 0M)) <= 0.000001M`

Reply:
On investigation, `decimal` is designed to be safe for such comparisons, unlike `float` and `double` so I can let it slide if you like.